lib/tasks/development.rb

```diff
@@ -82,7 +68,8 @@ def config_directory
   def dev_mounts
     [
       '-v', "#{config_directory}:/etc/ood/config",
-      '-v', "#{user.dir}/ondemand:#{user.dir}/ondemand"
+      '-v', "#{user.dir}/ondemand:#{user.dir}/ondemand",
+      '-v', "#{project_root}:/ondemand"
```

Comment:
I think the best practice that we should start telling folk is what's in DEVELOPMENT.md where you have the cloned repo in `~/ondemand/src`, so it would already get mounted in `~/ondemand` and preserve symlinks to say `~/ondemand/dev/dashboard`.

Reply:
Which is to say, I believe we already support what you're looking for if indeed have the ondemand source code cloned into `~/ondemand/<somewhere>`.

I think the main thing here is preserving symlinks so you have a dev dashboard within the container.

Reply:
Reverted this commit to keep old behavior